pkgs/top-level/python-packages.nix

```
@@ -19106,6 +19106,8 @@ self: super: with self; {
 
   x256 = callPackage ../development/python-modules/x256 { };
 
+  x3dh = toPythonModule (callPackage ../development/python-modules/x3dh { });
```

I don't think `toPythonModule` is required since this was neither created with `stdenv.mkDerivation` nor added to `pkgs/by-name` or `all-packages.nix` (see [docs](https://github.com/NixOS/nixpkgs/blob/322bec8709f2d6bd814af022b3ef9b10790cae85/doc/languages-frameworks/python.section.md#topythonmodule-function-topythonmodule-function)):

```suggestion
  x3dh = callPackage ../development/python-modules/x3dh { };
```